packages/dds/sequence/src/revertibles.ts

```diff
@@ -5,24 +5,23 @@
 
 /* eslint-disable no-bitwise */
 
-import { assert, unreachableCase } from "@fluidframework/core-utils";
+import { assert, unreachableCase } from "@fluidframework/core-utils/internal";
 import {
 	LocalReferencePosition,
 	MergeTreeDeltaOperationType,
 	MergeTreeDeltaRevertible,
 	MergeTreeDeltaType,
 	PropertySet,
 	ReferenceType,
-	SlidingPreference,
-	// eslint-disable-next-line import/no-deprecated
+	SlidingPreference, // eslint-disable-next-line import/no-deprecated
```

Comment:
Nit:  Comment moved, but still scoped to the correct "next-line"

Reply:
yuck/funny

Reply:
#20415